lighthouse-core/audits/metrics.js

```diff
@@ -118,6 +127,10 @@ class Metrics extends Audit {
       items: [metrics],
     };
 
+    if (artifacts.Visibility.some(event => event.state === 'hidden')) {
+      context.LighthouseRunWarnings.push(str_(UIStrings.warningVisibility));
```

Comment:
i think we want to issue a different error than NO_FCP if there was no fcp _and_ there was a visibility issue. introduce a TAB_BACKGROUNDED_DURING_AUDIT error-ish.
 
if we _do_ have FCP but have visibility issues, then this warning sgtm.